Name: Hide AmazonMP3 Store App
Version: 1.4.1.1-3
Author: v3ritas
Description: This patch will hide the AmazonMP3 Store App.

--- .orig/usr/palm/applications/com.palm.app.amazonstore/appinfo.json
+++ /usr/palm/applications/com.palm.app.amazonstore/appinfo.json
@@ -4,6 +4,7 @@
 	"main": "index.html",
 	"id": "com.palm.app.amazonstore",
 	"icon": "icon.png",
-	"splashicon": "icon-256x256.png"
+	"splashicon": "icon-256x256.png",
+	"visible": false,
 }
 
